README.md

```diff
@@ -1,41 +1,82 @@
-# IOTA Trinity Wallet
+<h1 align="center">
+  <br>
+  <a href="https://docs.iota.org/docs/wallets/0.1/trinity/introduction/overview"><img src="trinity.png"></a>
+</h1>
+
+<h2 align="center">The official IOTA wallet</h2>
+
+<p align="center">
+    <a href="https://docs.iota.org/docs/wallets/0.1/trinity/introduction/overview" style="text-decoration:none;">
+    <img src="https://img.shields.io/badge/Documentation%20portal-blue.svg?style=for-the-badge" alt="Developer documentation portal">
+</p>
+<p align="center">
+  <a href="https://discord.iota.org/" style="text-decoration:none;"><img src="https://img.shields.io/badge/Discord-9cf.svg?logo=discord" alt="Discord"></a>
+    <a href="https://iota.stackexchange.com/" style="text-decoration:none;"><img src="https://img.shields.io/badge/StackExchange-9cf.svg?logo=stackexchange" alt="StackExchange"></a>
+    <a href="https://raw.githubusercontent.com/iotaledger/trinity-wallet/develop/LICENSE" style="text-decoration:none;"><img src="https://img.shields.io/badge/License-Apache%202.0-green.svg" alt="Apache 2.0 license"></a>
+    <a href="https://dependabot.com" style="text-decoration:none;"><img src="https://api.dependabot.com/badges/status?host=github&repo=iotaledger/trinity-wallet" alt=""></a>
+</p>
+      
+<p align="center">
+  <a href="#about">About</a> ◈
+  <a href="#prerequisites">Prerequisites</a> ◈
+  <a href="#installation">Installation</a> ◈
+  <a href="#getting-started">Getting started</a> ◈
+  <a href="#supporting-the-project">Supporting the project</a> ◈
+  <a href="#joining-the-discussion">Joining the discussion</a> 
+</p>
+
+---
+
+## About
+
+This is the **official** IOTA wallet, which allows you to do the following:
+* Create password-protected accounts to store and access your seeds
+* Send transactions
+* Generate addresses
+* Read your balance and transaction history
+
+This is beta software, so there may be performance and stability issues.
+Please report any issues in our [issue tracker](https://github.com/iotaledger/trinity/issues/new).
 
-[![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=iotaledger/trinity-wallet)](https://dependabot.com)
+## Prerequisites
 
+To compile Trinity, you must have the following:
 
-This is the shared repository for the IOTA Trinity Mobile and Desktop wallets. The apps are based on React and built with React-Native (mobile) and Electron (desktop).
+* Node.js 8 or higher. We recommend the [latest LTS](https://nodejs.org/en/download/)
```

Comment:
```suggestion
* Node.js 10 or 11
```
Node 8 is probably fine to use but it has reached EOL. Some native components on desktop don't support Node 12 yet (we're awaiting fixes upstream) so I don't think we should say "or higher"

Reply:
Sounds good 👌 

I'll add a link to a downloads page